proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -300,3 +302,12 @@ message ConsumerAddrsToPrune {
   uint64 vsc_id = 2;
   AddressList consumer_addrs = 3;
 }
+
+// ConsumerRewardsAllocation is used to serialize the allocation of consumer chain rewards
```

Comment:
nit: could you expand this comment a bit? I don't understand it as written. it serializes the rewards, given as coin amounts, sent by consumer chains?

Reply:
Thanks for pointing this out. That's correct. It's used to track how many tokens were allocated to a consumer chain to the consumer rewards pool (module account).